src/server/common/model/application/application.test.js

```diff
@@ -25,20 +30,33 @@ const destinationDefaultState = {
   destinationType: 'dedicated-sale'
 }
 
+const biosecurityDefaultState = {
+  keptSeparately: 'yes'
+}
+
 describe('Application', () => {
   it('should create an Application instance from a valid state', () => {
     const state = {
       origin: originDefaultState,
       licence: licenceDefaultState,
-      destination: destinationDefaultState
+      destination: destinationDefaultState,
+      biosecurity: biosecurityDefaultState
+    }
+
+    const request = /** @type {Request} */ {
+      ...jest.requireActual('@hapi/hapi'),
+      yar: {
+        get: jest.fn().mockImplementation((key) => state[key])
+      }
     }
 
-    const application = ApplicationModel.fromState(state)
+    const application = ApplicationModel.fromState(request)
 
     expect(application).toBeInstanceOf(ApplicationModel)
 
     expect(application.tasks.origin).toBeInstanceOf(OriginSection)
     expect(application.tasks.licence).toBeInstanceOf(LicenceSection)
     expect(application.tasks.destination).toBeInstanceOf(DestinationSection)
+    expect(application.tasks.biosecurity).toBeInstanceOf(BiosecuritySection)
```

Comment:
We don't confirm that the state really makes it through to the various sections here - only that we've instantiated them.

Reply:
pushing  up a suggested test case for this in a min